mbq/tokens/decoder.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, certificate=None, allowed_audiences=None):
 
         self._allowed_audiences = set(allowed_audiences)
 
-    def decode(self, token):
+    def decode(self, token, verify_audience=True):
```

Comment:
There's an argument here that we shouldn't allow users of `mbq.tokens` to shoot themselves in the foot or knowingly avoid checking the audience because it's "easier." 

Another approach would be to provide a separate method `Decoder.decode_foreign_token` (with a better name) that has the same functionality but doesn't give the caller the impression that they can use it whenever they want a more relaxed decoder.

Reply:
Updated, totally open to suggestions on the name!